crest/Assets/Crest/Crest/Scripts/Spline/SplinePoint.cs

```diff
@@ -63,6 +68,14 @@ public override void OnInspectorGUI()
                 return;
             }
 
+            if (parent.TryGetComponent(out ISplinePointCustomDataSetup customData))
+            {
+                if (customData.AttachDataToSplinePoint(thisSP.gameObject))
+                {
+                    EditorUtility.SetDirty(thisSP.gameObject);
+                }
+            }
```

Comment:
Just a note. In the future we can use GetComponents instead to support more data types if that becomes a thing.

Reply:
Oh good point will do that

Reply:
Done, thanks